### DISCUSSION TITLE
Analytical approach to pick a new class

### DISCUSSION THREAD

**Discussion by Jroy21** [Hekili/hekili#1458] [Q&A]
Hi, I have a few younger players in my house and I'm trying to sort out which DPS class has the simplest rotation, more linear where the player can be less reactive and can get into a rhythm.

I know this is not directly related to Hekili but figured since the actual DPS priority and rotation are analyzed to the extreme and coded in this would be a good place to ask.

**Comment by Hekili** [comment]
I specifically avoid this topic in the context of the addon because I don't want people to pick what they play based on the addon or anything like that.

You might take a look at the SimC priority lists and analyze whether length/complexity of the priority there is any indicator of actual complexity.  I don't think it would be a perfect metric, because some things are fairly intuitive to a person and hard to explain to a computer, but it's the best answer I have to your question.

https://github.com/simulationcraft/simc/tree/shadowlands/profiles/Tier28

**Reply by Jroy21** [reply]
Thanks for taking the time to put in your opinion, greatly appreciated.

**Comment by samueldruker** [comment]
What Hekilii said for an analytic approach. There are lots of qualitative opinions out there, for example:

https://us.forums.blizzard.com/en/wow/t/easiest-class-to-play-for-beginners/149895/5

Analyzing the APL will have pitfalls. Some classes have some complicated corner cases or trinket usage or AoE that don't matter until you're in danger of climbing raider.io.

Consistently, I'd say DH, Beast Master, Frost Mage, Affliction 'lock and maybe Fury Warrior are all pretty straightforward. 

Definitely avoid Shaman, Fire Mage, Rogue, Feral, Balance...

Your Mileage May Vary :) Happy hunting!
